tap_hubspot_beta/streams.py

```diff
@@ -2111,4 +2111,43 @@ class UtmCampaignSummaryMonthlyStream(FormsSummaryMonthlyStream):
         th.Property("breakdowns", th.CustomType({"type": ["array", "string"]})),
         th.Property("start_date", th.DateType),
         th.Property("end_date", th.DateType),
-    ).to_dict()        
\ No newline at end of file
+    ).to_dict() 
+
+
+class GeolocationSummaryMonthlyStream(FormsSummaryMonthlyStream):
+    # https://legacydocs.hubspot.com/docs/methods/analytics/get-analytics-data-breakdowns
+    """Geolocation Summary Monthly Stream"""
+    name = "geolocation_summary_monthly"
+    path = "analytics/v2/reports/utm-campaigns/total"
```

Comment:
This `path` seems wrong, it is the same as `UtmCampaignSummaryMonthlyStream` based on the doc linked it should probably be 
```
analytics/v2/reports/geolocation/monthly
```

Reply:
You are right @hsyyid . Fixed it.